### ISSUE TITLE
Css Not importing from island in sub route

### ISSUE THREAD

**Issue by anatolysergov** [denoland/fresh#891]
I have a route in routes/subfolder/file.tsx 
When i attempt to use a island named Footer.tsx i get all the html elements display correctly but the css is not being imported. The css file is being imported using     

"<link rel="stylesheet" href={asset("/css/index.css")} /> "
or 
"link rel="stylesheet" href="/css/index.css" /> "

while the HTML is still rendering to the screen i am not getting the css along with the HTML elements. 

I am new to Fresh so if i am missing something please let me know. This could be an issue of my lack of understanding.

**Comment by codesculpture** [comment]
Can u give more information regarding this like file structure and actual source code of islands u mentioned and i suggest you to make sure the css on static folder if not works please provide more info

**Comment by codesculpture** [comment]
@anatolysergov if u reply then only the issue can be solved otherwise close this issue @hashrock

**Comment by anatolysergov** [comment]
I will update with an example, sorry had notifications off and missed the updates

**Comment by anatolysergov** [comment]
the file structure i am using for the island is Islands/Footer.tsx
within this island i am importing a css file using 
`export default function Footer() { 
    return (
        <>
            <Head>
            <link rel="stylesheet" href={asset("/css/footer.css")} />     
            </Head>
etc...`

The location i am attempting to use this is in 
routes/attorneys/attorneybio.tsx

I am using the footer tag in this route 
`export default function Home() {
    return (
      <>
        <div>
            <Head>
                <link rel="icon" href={asset("/blackSHLogo.svg")} />
                <link rel="stylesheet" href={asset("/css/lee.css")} />
            </Head>
            <Header/>
        </div>
        <Footer/>`  <----- The island in question

**Comment by anatolysergov** [comment]
It looks like when i import css/other static assets from a subfolder in the static folder [example Static/Css/Home.css] it is preventing me from retrieving static assets. Specifically when i am using the imports from an island

**Comment by anatolysergov** [comment]
Im not doing a very good job describing this so let me try again

In an island 'Islands/Footer.tsx' i have a static import to a css  file that is in Static/Css/index.css using the following import 

`<Head> <link rel="stylesheet" href={asset("/css/footer.css")} /> </Head>`  <--- Notice the subfolder css/cssfile.css

 -This island is able to import in any route that is the root of the routes folder Example: Routes/Home.tsx
 -When i attempt to use the island in a subroute Routes/Attroneys/Attorney1.tsx 
   I am not seeing the files being imported. It seems like when you are in a subroute it is not able to target subfolders within the static files. This includes imports with images and other files

**Comment by codesculpture** [comment]
Actually the i had tried the above scenario locally but it actually working. So, it seems very strange. Can you include your whole repo or full content of the files above mentioned ? 

And Let me say what i tried, 

Project Structure:
```bash 

├───islands
│       Test.tsx
│
├───routes
│   │   index.tsx
│   │
│   └───test
│           test1.tsx
│
├───static
│   │   logo.png
│   └───test
│           a.css
│
```

Test.tsx:
```tsx 
import { Head, asset } from '$fresh/runtime.ts'

export default function Test() {
        return <>
                <Head>
                        <link rel="stylesheet" href={asset("/test/a.css")} />
                </Head>
                <div class="name">Hello </div>
                </>
}
```
Here imported a sub foldered CSS.

test1.tsx 
 ```tsx 
import Test from '../../islands/Test.tsx';

export default function Main() {
        return <>

                <div>
                        <Test />
                </div>
        </>
}
```
<b> Note: test1.tsx is a subrouted (shown in project structure)</b>

The above was working perfect and fine.
And Your Question ```It seems like when you are in a subroute it is not able to target subfolders within the static files. This includes imports with images and other files```
<b>Ans: </b> We can import any static files regards of paths from any routes.
So Please provide the repo or source.

**Comment by anatolysergov** [comment]
https://github.com/anatolysergov/Skidmoreandhall

This is the repository, if you use the AttorneysView branch. Try to modify the Footer island changing the css location to css/footer then view the attorneys/lee route.

**Comment by codesculpture** [comment]
The issue u are facing is with path where is those are relative.
So when u mentioned
footer.css as 
```tsx 
asset("footer.css") 
```
 or

 ```tsx
 asset("css/footer.css")
 ``` 
it finds the css in its current path, that is when at [/attorneys]() it looks for current path that is at /css/footer.css 

But at /attorneys/lee/ it looks for /attorneys/css/footer.css so it responded with 404.

U need to just add a slash to the location of css 
**This is recommended**
```tsx 
asset("/css/footer.css")
 ```
or
```tsx 
asset("/footer.css")
 ```

The above PR where i fixed this issue
This is your solution😄

**Comment by anatolysergov** [comment]
ah man, thats a pretty brainfart moment. Thanks! I had an assumption it was me but figured i'd bring it up. Thanks again!
